app/components/NavModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const NavModalDialog = styled.div.attrs({
+  className: 'modal-dialog',
+})`
+  &&& {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+`;
+
+const NavModalContent = styled.div.attrs({
+  className: 'modal-content',
+})`
+  &&& {
+    background: transparent;
+    border: transparent;
+
+    & > div {
+      border: inherit;
+    }
+  }
+`;
+
+const ModalCloseX = styled.span`
+  color: #ddd;
+  font-size: 3.25rem;
```

Comment:
I want to avoid using custom font sizes per component, and instead standardize our text size either through const variables or specific components e.g. `<TextMedium>` etc. I think it's fine here because the modal is a special case but just something to keep in mind for the future

Reply:
I agree. Seemed OK here since the x is treated more like a graphic (could possibly make it its own component at some point).